DevIncubator-Autopark/VehicleHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal static class VehicleHelper
+    {
+        public static void ShowAllVehicles(IReadOnlyList<Vehicle> vehicles)
```

Comment:
Why is 'All' in the name of method? You could not know is it all or not all.

Reply:
fixed